templates/package/view_package.html.twig

```diff
@@ -319,6 +324,33 @@
                 </div>
             {% endif %}
 
+            {% if is_granted('transfer_package', package) %}
+                <div class="row">
+                    {{ form_start(transferPackageForm, {
+                        attr: { id: 'transfer-package-form', class: 'col-sm-6 col-md-3 col-md-offset-9 ' ~ (show_transfer_package_form|default(false) ? '': 'hidden') },
+                        action: path('transfer_package', { 'name': package.name })
+                    }) }}
+                        <div>
+                            <h4>Transfer Ownership</h4>
+                            <div class="maintainers-collection-wrapper">
+                                {{ form_errors(transferPackageForm.maintainers) }}
+                                <ul class="maintainers-list" data-prototype="{{ form_widget(transferPackageForm.maintainers.vars.prototype)|e('html_attr') }}">
+                                    {% for maintainerField in transferPackageForm.maintainers %}
+                                        <li>
+                                            {{ form_errors(maintainerField) }}
+                                            {{ form_widget(maintainerField) }}
+                                        </li>
+                                    {% endfor %}
+                                </ul>
+                                <button type="button" class="btn btn-info btn-sm add-maintainer-item">Add Maintainer</button>
+                            </div>
+                            {{ form_rest(transferPackageForm) }}
+                            <input class="btn btn-block btn-success btn-lg" type="submit" value="Transfer Ownership" />
```

Comment:
add an onclick confirm just in case? its a very important action behind the scene